app/components/home/delegation-card.tsx

```diff
@@ -7,115 +7,165 @@ import { useCalculateFee } from '@hooks/use-calculate-fee';
+    revocationFee
   );
 
   if (!delegationStatus.delegated) return null;
 
   return (
-    <Flex
-      flexDirection="column"
-      mt="extra-loose"
-      borderRadius="8px"
-      boxShadow="0px 1px 2px rgba(0, 0, 0, 0.04);"
-      border={`1px solid ${color('border')}`}
-      minHeight="180px"
-    >
-      <Box>
-        <Flex mt="loose" justifyContent="center">
-          <DelegatedIcon size="44px" />
-        </Flex>
-        <Text
-          display="block"
-          color={color('text-caption')}
-          textStyle="caption"
-          mt="tight"
-          textAlign="center"
-        >
-          You've delegated up to
-        </Text>
-        <Flex justifyContent="center" mt="tight">
-          <Text textStyle="body.large.medium" fontSize="24px">
-            {toHumanReadableStx(delegationStatus.amountMicroStx.toString())}
-          </Text>
-        </Flex>
-        <Box mr="2px">
-          <Flex flexDirection="column" alignItems="center" mt="base-tight" mb="base">
-            <Text textStyle="caption" color={color('text-caption')}>
-              Delegated to
-            </Text>
-            <Text fontSize="13px" mt="tight" color={color('text-title')}>
-              {truncateMiddle(delegationStatus.delegatedTo, 6)}
-            </Text>
-            {delegationStatus.deadDelegation && (
-              <Flex flexDirection="column" alignItems="center" mt="base">
-                <ErrorLabel mt="1px" px="loose">
-                  <ErrorText>
-                    Your delegation has expired. Your pool is no longer able to stack on your
-                    behalf. Revoke your delegation and reinitiate if you wish to continue Stacking.
-                  </ErrorText>
-                </ErrorLabel>
-              </Flex>
-            )}
-          </Flex>
-        </Box>
-        <Box borderTop={`1px solid ${color('border')}`} py="extra-tight" px="extra-tight">
-          {hasSufficientBalanceToCoverFee ? (
-            <Button
-              variant="link"
-              border={0}
-              py="base-tight"
-              px="base"
-              textStyle="body.small"
-              color={color('text-body')}
-              pointerEvents={
-                hasPendingRevokeCall || !hasSufficientBalanceToCoverFee ? 'none' : 'all'
-              }
-              cursor={
-                hasPendingRevokeCall || !hasSufficientBalanceToCoverFee ? 'not-allowed' : 'unset'
-              }
-              onClick={() => dispatch(homeActions.openRevokeDelegationModal())}
-            >
-              {hasPendingRevokeCall ? 'Currently revoking STX' : 'Revoke delegation'}
-            </Button>
-          ) : (
+    <InfoCard minHeight="180px" mt="extra-loose" px={['loose', 'extra-loose']}>
+      <Flex mt="loose">
+        <img
+          src={pooledStackingImg}
+          alt="Colourful illustration of a diving board protruding out of a blue hole"
+        />
+      </Flex>
+
+      {!delegationStatus.deadDelegation && (
```

Comment:
I keep seeing `delegation`, what does that mean?

Reply:
Delegation is the type of stacking you do in a _"pool"_ with others. We're generally using the term pooled stacking now, but the technical term is delegated stacking

Reply:
Very helpful, thank you!